scripts/globals/dealer_moogle.lua

```diff
@@ -0,0 +1,545 @@
+-----------------------------------
+-- Dealer Moogles & Kupon Global
+-- https://www.bg-wiki.com/ffxi/Category:Mog_Bonanza
+-----------------------------------
+require("scripts/globals/items")
+require("scripts/globals/keyitems")
+require("scripts/globals/zone")
+-----------------------------------
+xi = xi or {}
+xi.dealerMoogle = xi.dealerMoogle or {}
+
+local csidLookup =
+{
+    [xi.zone.CHOCOBO_CIRCUIT] = { 416, 417 },
+    [xi.zone.PORT_SAN_DORIA]  = { 790, 791 },
+    [xi.zone.PORT_BASTOK]     = { 398, 399 },
+    [xi.zone.PORT_WINDURST]   = { 856, 857 },
+}
+
+-- TODO: Where is Kupon A-E +2
+-- TODO: Where is Kupon AW-GeIV
+-- TODO: Where is Kupon AW-UWIII
+
+-- WARNING: These items cannot be customised, and must be in the order
+-- WARNING: they appear in the in-game menus!
```

Comment:
```
-- WARNING: 
-- These items cannot be customized, 
-- and must be in the order they appear in the in-game menus!
```
_suggested_ revision. not a big deal but my brain had to read twice do to line break placement and double "warning"